README.md

```diff
@@ -30,8 +30,6 @@ $ yarn lint:prettier
 
 ## Deployment
 
-This project is deployed and hosted on Vercel, running on the **abakus-nutu** Vercel user/github team.
+This project is deployed and self-hosted with Coolify.
 
-When you are ready to a new verison to **fondet.abakus.no**, you merge your branch with master, and `vercel` will run a `--prod` version. This will result in a new version of `fondet.abakus.no`.
-
-[![](./public/powered-by-vercel.svg)](https://vercel.com/?utm_source=team_XD0ckB9pO5pVMJVVqyDd9zDp&utm_campaign=oss)
+When you are ready to a new verison to **fondet.abakus.no**, you merge your branch with master, and `coolify` will run a `prod` version. This will result in a new version of `fondet.abakus.no`.
```

Comment:
```suggestion
When you are ready to deploy a new version to **fondet.abakus.no**, you merge your branch with master, and `coolify` will run a `prod` version. This will result in a new version of `fondet.abakus.no`.
```